### DISCUSSION TITLE
Todo List and Pull requests.

### DISCUSSION THREAD

**Discussion by aprilthepink** [jagrosh/MusicBot#603] [Questions] [answered]
What happened to the todo list and why  are some pull requests just left unanswered? Just asking don't want to set pressure on you guys :)

**Comment by jagrosh** [comment, marked as answer]
The projects list was removed because it felt like a redundant and less-maintained version of the enhancement label (https://github.com/jagrosh/MusicBot/labels/enhancement), and the Discussions feature integrates better with the Issues list.

Some pull requests are unanswered simply because I haven't had time to review them yet, haven't decided if they are features that should be added at this time, or they haven't been updated since I've requested changes.

**Comment by aprilthepink** [comment]
ok thx and I did only mean thinks you haven't requested changes yet ;)
